exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,20 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0){
+    return true;
+  }
```

Comment:
@rahwafesseha Maira and I talked about this and we agreed that it's okay not to have `else` :)

An advanced way of writing this could be. Can you guess why it's still correct?
```
return number < 0;
```